src/store/Flowers.java

```diff
@@ -0,0 +1,10 @@
+package store;
+
+public class Flowers {
```

Comment:
make abstract class with field price, and add constructor with param in it. better name "Flower"